qiita_pet/templates/redbiom.html

```diff
@@ -208,18 +208,31 @@
         <ul>
           <li>
             Find all samples in which the word infant exists, as well as antibiotics,
-            where the infants are under a certain number of days old:
-            "infant &amp; antibiotics where age_days < 30"
+            where the infants are under a year old:
```

Comment:
Remember that this is HTML so it will not render as it's written. If you want a new line you will need to use `<br/>`, if you want to create a list you could do: 
```html
<ul>
   <li>element 1</li>
   <li>element 2</li>
</ul>
```
and please use `&amp;` for all the `&`